sdk/core/amqp-common/rollup.config.js

```diff
@@ -11,6 +11,11 @@ if (!process.env.ONLY_BROWSER) {
 
 if (!process.env.ONLY_NODE) {
   inputs.push(base.browserConfig());
+  inputs.push(base.browserConfig({ production: true }));
+}
+
+if (process.env.BROWSER_TEST) {
```

Comment:
`BROWSER_TEST` is not an env variable set by any of the npm scripts, when do we expect to pass this `if` condition?